system/host_cpu.tf

```diff
@@ -0,0 +1,47 @@
+resource "datadog_monitor" "cpu_average" {
+  count   = "${var.monitor_enabled}"
+  name    = "${module.label.id}"
+  type    = "${var.alert_type}"
+  message = "High CPU usage last ${var.cpu_time} on host: ${data.aws_instance.monitored.instance_id} with IP: ${data.aws_instance.monitored.public_ip}"
```

Comment:
I'm not crazy about the name `cpu_time` because that usually refers to the amount of time consumed on a CPU. This is referring to the (period,window,duration) during which the CPU exceeded the critical threshold value.

Reply:
I would like the message to look more like:

"CPU usage exceeded ${var.cpu_critical_threshold_value} for the last ${var.cpu_time} on {host}"